README.md

```diff
@@ -5,14 +5,16 @@ The IMPAnalyzer now supports the following option:
 java -cp ./build/libs/*: com.netflix.imflibrary.app.IMPAnalyzer <directory> [-a <APPSTRING> | --application <APPSTRING>]
 ```
 ```
-options:            
--a | --application <APPSTRING>      IMF Application to test against, supported values for <APPSTRING> are app2or2E, app5 or all (default: app2or2E)
+Options:
+-a | --application <APPSTRING>      IMF Application to test against, supported values for <APPSTRING> are app2or2E and app5
 ```
 
 Notes:
 1. The option "-a | --application" is supported for the application ```IMPAnalyzer``` only.
 
-2. By default, Photon tests against the App #2/#2E specification.
+2. By default, Photon tests against the Application specified in the ```ApplicationIdentification``` element of a CPL.
+
```

Comment:
What if there are more than one Application listed in `ApplicationIdentification`

Reply:
Photon reads the ApplicationIdentification element of a given CPL only, if the list size equals to one:
https://github.com/Netflix/photon/blob/aa80dc816200fd80877931b477abded72d2bba3b/src/main/java/com/netflix/imflibrary/st2067_2/CompositionModel_st2067_2_2016.java#L195-L197

If the size of the ApplicationIdentification list is greater than one, Photon throws an error and tests against the App2E spec by default:
https://github.com/Netflix/photon/blob/148fec9bcab8cee457cd9034f0c5e66d0579e563/src/main/java/com/netflix/imflibrary/st2067_2/ApplicationCompositionFactory.java#L102-L106

As a consequence, the proposed option `--application APPSTRING` will lead to no additional error message if `APPSTRING == app2or2E`, but will throw an additional error if `APPSTRING == app5`.

I think this is overall a reasonable behaviour for this exotic case.

Reply:
Do you think we can support a list of application here (splitted via comma):
`--applications app2,app5`
because with next application it will become complexe to add all combination (I have submitted to support TSP 2121-1, TSP 2121-4 can come this year and App4 too).

So internally we can have an array of `ApplicationCompositionType`.
What do you think ?

Reply:
> If the size of the ApplicationIdentification list is greater than one, Photon throws an error and tests against the App2E spec by default:

Ok. It sounds like an issue beyond this pull request.